### ISSUE TITLE
no firmware.

### ISSUE THREAD

**Issue by youxiaojie** [roybaer/usb_floppy_adapter#1]
no firmware.

**Comment by roybaer** [comment]
The repository does not contain any firmware, because there is no firmware for the device, yet.
The only thing that exists is an stm32duino sketch that reads a number from a serial connection and moves the drive head to that position.

**Comment by roybaer** [comment]
It should be possible to adapt the Greaseweazle software.  See https://github.com/keirf/Greaseweazle/issues/4.

**Comment by gilmod714** [comment]
it has been a few years .. has this project been abandoned?! .. or is there a firmware in the making ?! .. 

I see this device being very helpful for people .. especially for people who will try to integrate it into a cheap "blue pill"

Thank you :)

**Comment by roybaer** [comment]
Yes, for the time being, this has been abandoned. If and when I return to it, I will probably redesign the PCB layout to adopt the exact pinout used by Greaseweazle for "blue pill" adapter boards. There were a few issues with my choice of µC pins and the pull-up resistors appear to be largely unnecessary, too, because the FDD bus uses TTL compatible levels, anyway.
I did not pursue this any further after finding out that my USB FDD actually supports 720 KiB DD disks once they have been formatted elsewhere, and that was my primary use case.

Bottom line: You can already use a "blue pill" board with Greazeweazle if you don't care about a pretty circuit board.
At this point, I'd rather redesign the board to use unmodified firmware and not do it the other way around.
